indico/htdocs/js/indico/jquery/categorypicker.js

```diff
@@ -0,0 +1,123 @@
+            var opt = self.options;
+
+            self.searchInput = $('<input>', {
+                class: 'js-search-category',
+                type: 'text',
+                placeholder: $T.gettext("Search")
+            });
+            self.categoryList = $('<ul>').addClass('group-list');
+            self.categoryNavigator = $('<div>')
+            .addClass('category-navigator i-box just-group-list with-hover-effect fixed-height')
+            .append($('<div>').addClass('i-box-content'))
+            .append(self.categoryList);
+
+            buildCategoryList(opt.initialCateg, self.categoryList);
+
+            self.element.append(self.searchInput).append(self.categoryNavigator);
+
+            function goToCategory(categoryId, $categoryList) {
+                $('.category-navigator .item').animate({
+                    height: 0,
+                    paddingTop: 0,
+                    paddingBottom: 0,
+                    border: 0
+                }, {
+                    complete: function() {
+                        $(this).remove();
+                    }
+                });
+                buildCategoryList(categoryId, $categoryList);
+            }
+
+            function buildCategoryList(categoryId, $categoryList) {
+                $.ajax({
+                    url: build_url(Indico.Urls.Categories.info, {categId: categoryId}),
+                    method: 'POST',
+                    dataType: 'json',
+                    error: function(data) {
+                        handleAjaxError(data);
+                    },
+                    success: function(data) {
+                        if (data.category) {
+                            var category = $('<li>', {
+                                class: 'item parent-category',
+                                data: {
+                                    id: data.category.id
+                                }
+                            }).append($('<span>', {
+                                class: 'title',
+                                text: data.category.title,
+                            }));
+                            var breadcrumb = $('<span>').addClass('breadcrumb');
+                            for (var i = 0; i < data.category.breadcrumb.length; i++) {
+                                breadcrumb.append($('<a>', {
+                                    text: data.category.breadcrumb[i].title,
+                                    data: {
+                                        id: data.category.breadcrumb[i].id
+                                    },
+                                    href: '#'
+                                }).on('click', function() {
+                                    goToCategory($(this).data('id'), $categoryList);
+                                }));
+                                if (i != data.category.breadcrumb.length - 1) {
+                                    breadcrumb.append($('<span>', {
```

Comment:
What about using an `ul` and `li` items for the breadcrumbs? Then you can use `li:not(:last-child):before` to add the separator inbetween the items